doc/network.tex

```diff
@@ -1713,16 +1738,16 @@ \subsection{Setup of a Connection with the Mux-Control Protocol}
 and the server replies either with a \InitRsp~message containing the negotiated version
 or a \InitFail~message.
```

Comment:
It is true that _version negotiation_ is run exactly once, and in the code right now there is nothing else defined over Mux-Control. However clean shutdown (Which is mentioned in section 4.2 Setup, Shutdown and Management of Connections) you mention "Handle a shutdown request from the peer." which will be transmitted over Mux-control once it is implemented.

Reply:
So it is better to comment out the hole Section ?
If the only info is that the mux-control is one once at the beginning of a connection
an nothing else is in the Section then maybe comment out the section alltogether ?

Reply:
I think that simply changing it from talking about Mux-control in general to version negotiation within Mux-control would be enough.